### ISSUE TITLE
Feature request - Epidemic Calculator

### ISSUE THREAD

**Issue by spongioblast** [gabgoh/gabgoh.github.io#5]
Hi,
Would you mind adding the function of "Max treatment capacity", eg ventilators available for Covid-19 cases, and a death-rate for patients which can't be treated due to being over the capacity of of the hospital infrastructure in the country? Example attached.
[Coronavirus Modeling.xlsx](https://github.com/gabgoh/gabgoh.github.io/files/4349428/Coronavirus.Modeling.xlsx)

Thanks,
Adrian

**Comment by siriniok** [comment]
Yes, it's very important, as CFR is changing dynamically depending on the treatment capacity (number of intensive care units with ventilators). Without this parameter you can't get an accurate model.

**Comment by Matthias-Fauconneau** [comment]
https://github.com/neherlab/covid19_scenarios
